src/utils.js

```diff
@@ -33,6 +33,9 @@ export const degrees = (a) => {
 };
 
 export const isPressed = (evt) => {
+    if (evt.type === 'pointerdown' || evt.type === 'pointermove') {
```

Comment:
Is it possible to have a `evt.type === 'pointerdown'` here? I don't see it.

Also, following that logic, maybe it will be the same for `touchstart` and `touchmove` maybe?

Reply:
Sorry for late reply. When I tested with iPhone 11, I have confirmed 'pointerdown' event through here.
However, I think this is not fundamental solution, is just workaround.
So could I close this PR?

Reply:
don't close it, I think this is a good solution.
I'll take over if you don't have time for this, no worries.

Reply:
Ok, I got it. 
I don't have enough time for debugging this now. 

I'm working on some game project with nipplejs, and I got another bug with iOS 13 on iPhone SE: Sometimes joypad will not be destroyed, remains show up without disappearing even if I  lift up my finger.

So I fork this, and I'm using this branch code.
https://github.com/umeruma/nipplejs/commit/3be863857c55793c0e79129f68d5df4de5c79c1a